lib/error.ex

```diff
@@ -2,6 +2,18 @@ defprotocol AshGraphql.Error do
   def to_error(exception)
 end
 
+defimpl AshGraphql.Error, for: Ash.Error.Changes.InvalidChanges do
+  def to_error(error) do
+    %{
+      message: error.message,
+      short_message: error.message,
+      vars: Map.new(error.vars),
+      code: Ash.ErrorKind.code(error),
+      fields: [error.field]
```

Comment:
I think we want something like `List.wrap(error.field) ++ error.fields` in this case, IIRC that error can have a `field` or a `fields`.

Reply:
I just checked but it looks like it only has fields option
>   def_ash_error([:fields, :message, :validation], class: :invalid)

Reply:
I believe they all have a `field` key by default, but that this error specifically adds an additional `fields` field. Confusing, I know.

Reply:
But what you have is fine :) IF you're using this error you're expected to set `fields` and not `field`

Reply:
But what you have is fine :) IF you're using this error you're expected to set `fields` and not `field`